### ISSUE TITLE
ssb vs. sb

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [nmfs-fish-tools/data_dictionary#8]
It seems weird to me that we use spawning stock biomass but do not add the "stock" modifier to any other term, e.g., we use age 3+ biomass not age 3+ stock biomass. Should we advocate for spawning biomass instead of spawning stock biomass?

**Comment by chantelwetzel-noaa** [comment]
I totally agree.  It feels like "SSB" and "spawning stock biomass" is a legacy term that is no longer used when describing results.  I would be okay attempting to move away from this but we may want to ask around a bit to understand why spawning stock biomass was initial used.

**Comment by kellijohnson-NOAA** [comment]
Sounds good. I will ask Hastie, Jon Brodziak, and Chris Legault. Let me know if you think of others. The [Australian glossary](https://www.awe.gov.au/abares/research-topics/fisheries/fishery-status/glossary-acronyms#abbreviations) just lists "spawning biomass" so I am assuming this will be fine.

**Comment by kellijohnson-NOAA** [comment]
@Bai-Li-NOAA can you quantify the use of spawning biomass (SB) versus spawning stock biomass (SSB) in the stock assessments?

**Comment by Bai-Li-NOAA** [comment]
@kellijohnson-NOAA. Yes. Will do that today and share the outputs with you!
